Sources/WebSocketKit/WebSocketClient.swift

```diff
@@ -44,7 +45,11 @@ public final class WebSocketClient {
         case .shared(let group):
             self.group = group
         case .createNew:
+            #if canImport(Network)
+            self.group = NIOTSEventLoopGroup(loopCount: 1, defaultQoS: .default)
+            #else
             self.group = MultiThreadedEventLoopGroup(numberOfThreads: 1)
+            #endif
```

Comment:
I would leave this part out - there are legitimate reasons to want to default to a plain MTELG even when Network.framework is available (most often on macOS). The `.shared` case provides ample ability to specify an appropriate ELG if desired. (Worth noting, I suggest it as much for consistency with other packages that do the same thing as for any other reason.)

Reply:
@gwynne I see, thanks! Will remove it in next commit 👍